packages/frontend/src/pages/ManageEventsPage/components/EventCard/EventCard.tsx

```diff
@@ -47,10 +52,10 @@ export const EventCard = ({ event }: { event: IEvent }) => {
         <div
           className={getClassesList(
             css.evenCardRegistration,
-            isRegistrationOpen ? css._open : css._closed
+            !isRegistrationOpen ? css._open : css._closed

```

Comment:
this would be better (one less operation): `isRegistrationOpen ? css._closed : css._open`

Reply:
Done